app/controllers/foreman_plugin_template/react_controller.rb

```diff
@@ -0,0 +1,17 @@
+module ForemanPluginTemplate
+  class ReactController < ::ApplicationController
+    def index
+      render 'foreman_plugin_template/layouts/react', :layout => false
```

Comment:
can you instead add a default layout in `app/views/layout/foreman_plugin_template/application.html.erb`
and than set it on line 3 as the default layout: `layout "foreman_plugin_template/application"`

this will reduce the need to call the render method on each action as it will automatically render ` app/views/foreman_plugin_template/react/index.html.erb `